AppManifest.json

```diff
@@ -23,6 +23,11 @@
                     "image": "ghcr.io/boschglobal/kuksa.val/databroker",
                     "version": "0.0.2"
                 },
+                {
```

Comment:
This should not be in the AppManifest.json. Why was this added?

Reply:
Removed as discussed